src/safir/slack/blockkit.py

```diff
@@ -338,14 +338,14 @@ def from_exception(
             )
         else:
             message = f"{type(exc).__name__}: {str(exc)}"
-            if isinstance(exc, RequestError):
+            try:
                 return cls(
                     message,
                     method=exc.request.method,
                     url=str(exc.request.url),
```

Comment:
Are we using exception handling here instead of a `hasattr` test because sometimes there is a `request` attribute but it's `None`?

The main thing I might recommend is just adding a comment that not all RequestError exceptions have a `request`.

Reply:
Yup, exactly. What happens is that it gets set to `None` in the constructor and then later exception handling adds it via attribute assignment, but that attribute assignment doesn't always happen. I'll add a comment explaining that.

Reply:
And hm, maybe I should just explicitly check for `None`? Using an exception handler is more comprehensive, but kind of confusing.

Reply:
Oh, right, now I remember: you can't check for `None` because the actual data is hidden, and `request` is a property that raises `RuntimeError` if the request hasn't been set. I'll add that to the comment as well.